vcd/resource_vcd_vapp_network.go

```diff
@@ -233,7 +236,7 @@ func resourceVappNetworkRead(d *schema.ResourceData, meta interface{}) error {
 			d.Set("dns2", c.IPScopes.IPScope.DNS2)
 			d.Set("dnsSuffix", c.IPScopes.IPScope.DNSSuffix)
 		}
-		d.Set("guestVlanAllowed", c.GuestVlanAllowed)
```

Comment:
How did this work before? Was there a different bug?

Reply:
Nothing was overriden, nothing was querring.